doc/script_commands.txt

```diff
@@ -6481,15 +6481,28 @@ Size is 0 = normal 1 = small 2 = big.
 *addtimer(<ticks>, "NPC::OnLabel"{, <account id>})
 *deltimer("NPC::OnLabel"{, <account id>})
 *addtimercount("NPC::OnLabel", <ticks>{, <account id>})
+*gettimer(<type>{, <account id>{, <event>}})
 
 These commands will create, destroy, and delay a countdown timer - 
-addtimer() to create, deltimer() to destroy and addtimercount() to delay
-it by the specified number of ticks. For all three cases, the event label 
-given is the identifier of that timer. The timer runs on the character 
-object that is attached to the script, and can have multiple instances.
-If <acccount id> is passed, this player will be used instead.
-When the label is run, it is run as if the player that the timer runs on 
-has clicked the NPC.
+addtimer() to create, deltimer() to destroy, addtimercount() to delay
+it by the specified number of ticks, and gettimer() to get misc info.
+For all three cases, the event label given is the identifier of that timer.
+The timer runs on the character object that is attached to the script,
+and can have multiple instances. If <acccount id> is passed, this player
+will be used instead. When the label is run, it is run as if the player
+that the timer runs on has clicked the NPC.
+
+valid <type> for gettimer() are:
+
+(0) TIMER_COUNT
+	Will return the total number of timers for the specified or
+	attached player. Can be filtered by <event>.
+(1) TIMER_FIRST_TICK
+	Will return the number of ticks until the next timer runs
+	for the specified or attached player. Can be filtered by <event>.
+(2) TIMER_LAST_TICK
```

Comment:
no this is the amount of ticks remaining until the last event runs so given this:
```c++
addtimer(700, "event");
addtimer(900, "event");
addtimer(200, "event");
```
the amount of ticks until `"event"` gets called for the last time would be `900` (while `REMAINING_TICK` would imply the amount of ticks until it is called next, which would be `200`)

- - - 
in short, `FIRST_TICK` would be `200` and `LAST_TICK` would be `900`, which correspond to the first and last times the event is invoked